query/query_manager.rs

```diff
@@ -80,6 +80,9 @@ impl QueryManager {
         source_query: &str,
     ) -> Result<Pipeline<Snapshot, ReadPipelineStage<Snapshot>>, Box<QueryError>> {
         event!(Level::TRACE, "Running read query:\n{}", query);
+        let mut query_profile = QueryProfile::new(tracing::enabled!(Level::TRACE));
```

Comment:
Query profiles are currently detatched from transcation profiles, though they could be linked!